src/plugins/saved_objects_finder/public/finder/saved_object_finder.tsx

```diff
@@ -45,8 +39,8 @@ export interface SavedObjectMetaData<T = unknown> {
   getTooltipForSavedObject?(savedObject: SavedObjectCommon<T>): string;
   showSavedObject?(savedObject: SavedObjectCommon<T>): boolean;
   getSavedObjectSubType?(savedObject: SavedObjectCommon<T>): string;
+  /** @deprecated doesn't do anything, the full object is returned **/
```

Comment:
I'll try to spend some time and see if it will be easy and worth it adding this back https://github.com/elastic/kibana/issues/163043 , didn't want to delete, as there is a chance we can bring this functionality back without changing the component outside outside